### PR TITLE
8302806: (fs) Remove com.sun.nio.file.SensitivityWatchEventModifier

### PR DESCRIPTION
**Pull request by bplb**
This enum is not used in the JDK and did not appear in external source code searches.

---------
### Progress
- [ ] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [ ] Change requires CSR request [JDK-8303097](https://bugs.openjdk.org/browse/JDK-8303097) to be approved

### Issues
 * [JDK-8302806](https://bugs.openjdk.org/browse/JDK-8302806): (fs) Remove com.sun.nio.file.SensitivityWatchEventModifier
 * [JDK-8303097](https://bugs.openjdk.org/browse/JDK-8303097): (fs) Remove com.sun.nio.file.SensitivityWatchEventModifier (**CSR**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/12626/head:pull/12626` \
`$ git checkout pull/12626`

Update a local copy of the PR: \
`$ git checkout pull/12626` \
`$ git pull https://git.openjdk.org/jdk pull/12626/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 12626`

View PR using the GUI difftool: \
`$ git pr show -t 12626`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/12626.diff">https://git.openjdk.org/jdk/pull/12626.diff</a>

</details>

### CODE REVIEW

**Comment by AlanBateman** [comment]
This API should probably have been deprecated many releases ago. It's the jdk.unsupported module so it may be possible to remove without first deprecating.

The PR is missing the implementation and test changes.

**Review by AlanBateman — COMMENTED** [review]
You need to remove the SENSITIVITY_xxx constants from sun.nio.fs.ExtendedOptions and that will should help identify the code in the polling WatchService implementation that need to be removed.

File: src/java.base/linux/classes/sun/nio/fs/LinuxWatchService.java [hunk 1]
```diff
@@ -226,14 +226,11 @@ Object implRegister(Path obj,
             }
 
             // no modifiers supported at this time
-            for (WatchEvent.Modifier modifier : modifiers) {
+            if (modifiers != null) {
```
Comment (AlanBateman) [inline src/java.base/linux/classes/sun/nio/fs/LinuxWatchService.java:229]:
```suggestion
            if (modifiers.length > 0) {
```
Same thing in PollingWatchService.

**Comment by bplb** [comment]
Withdrawing this PR for now. The most recent version of the commits is attached to the issue.

**Pull request closed without merging**